decidim-core/lib/decidim/core/test/shared_examples/has_attachments.rb

```diff
@@ -81,7 +81,7 @@
     it "shows them ordered" do
       find("li [data-controls='panel-documents']").click
       within "#panel-documents" do
-        expect(translated(first_document.title, locale: :en)).to appear_before(translated(last_document.title, locale: :en))
+        expect(escape_translated(first_document.title).gsub("&quot;", "\"")).to appear_before(escape_translated(last_document.title).gsub("&quot;", "\""))
```

Comment:
Any idea why is this happening? Maybe there's like a double escape or something like that?

Reply:
i have not investigated where this is being generated. But, i would tackle it after all the work in scope is done.

Reply:
Keeping track of this in #12336